php/class-coauthors-wp-list-table.php

```diff
@@ -276,8 +278,8 @@ function extra_tablenav( $which ) {
 			echo '<option value="' . esc_attr( $key ) . '" ' . selected( $this->active_filter, $key, false ) . '>' . esc_attr( $value ) . '</option>';
 		}
 		echo '</select>';
+	    submit_button( __( 'Filter', 'co-authors-plus' ), 'secondary', false, false );
```

Comment:
Similarly, here it looks like 1 tab + 4 spaces. Maybe 2 tabs for it.

Reply:
See https://github.com/Automattic/Co-Authors-Plus/pull/768#discussion_r603798628